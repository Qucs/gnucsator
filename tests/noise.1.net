# Qucs

Vnoise:N0 _net0 gnd u=1
Vnoise:N1 _net1 _net0 u=1
Vnoise:N2 _net2 gnd u=2
Vnoise:N3 _net3 gnd u=1 e=2
.AC:AC1 Type="lin" Start="1 MHz" Stop="2 MHz" Points="3" Noise="yes"
