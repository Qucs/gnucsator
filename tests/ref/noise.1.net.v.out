
module main ();
//# Qucs 
Vnoise #(.u(1)) N0 (.p(_net0),.n(0));
Vnoise #(.u(1)) N1 (.p(_net1),.n(_net0));
Vnoise #(.u(2)) N2 (.p(_net2),.n(0));
Vnoise #(.u(1),.e(2)) N3 (.p(_net3),.n(0));
endmodule // main

